consumer-api/consumer_api_v1.0.json

```diff
@@ -654,7 +654,7 @@
             },
             "station": {
                 "title": "Root Type for Agency",
-                "description": "An agency which has data stored in the TDEI system. Includes agency_id and associated agency_name. ",
+                "description": "An station which has data stored in the TDEI system. Includes station_id and associated station_name. ",
```

Comment:
A station??

Reply:
There are a number of issues with this line- 
The identifier, "description" is an ambiguous tag. We should be more specific in the nomenclature.
The value description is non specific as well. 
I suggest we replace this line with 
"agency_id": "agency_id",
"agency_name": "agency_name",
"stop_id": "stop_id" (corresponding to stops.txt)
"stop_name": "stop_name"
"agency_ids": "agency_id_list",
"agency_names": "agency_names_list",
"stop_ids": "stop_id_list" (corresponding to stops.txt)
"stop_names": "stop_names_list"

(the last 4 lines has to do with the potential one-to-many relationship between the pathways file for a specific station and multiple agencies operating them)

Reply:
@uwtcat - description is the tag specified by openapi, that is not our choice. Also, it's not clear in the json, which is complex, but there is both a description and an example of the json that would be used to represent the station. This is all more clear - I think - if you look at the apicurio version of the API.

In any case, the example json in the json is (for station):

{
    "station": {
        "agency_id": 1,
        "agency_name": "King County Metro",
        "station_id": 4,
        "station_name": "Northgate"
    }
}

This object is for representing a particular station. I didn't realize that multiple agencies can manage a single station. Is it possible to designate a single agency as 'primary' agency? For pathways, seems that many stations would have a single primary agency. Is that not true? I think we will need a primary agency for the purpose of conflict resolution also. 

@susrisha yes, it should be A station - the grammar in the descriptions is not clean - I haven't taken the time to clean it up. Also to @uwtcat 's point, description could be more descriptive.

Reply:
@uwtcat , thinking a bit, I'd be ok with a list of agency_ids, no agency names. And the relationship between stops and station will be represented by having a stops object with a stationid in it to capture the relationship. I didn't create a stops object in the API because we are just providing a full pathways file which will have stops.txt and those relationships inside of it, so we don't need to expose the stops in the API. Let me know if you want to discuss further.

Reply:
@kristinann thank you for pointing out the "description" tag is required. I think I better understand this field even though I'm pressed to understand how it would be used. I suppose, for instance, a consumer might put it on an app when the user clicks on a particular station and a picture of the structure and short "description" comes up? In this case,  I guess we should keep it in, but I may put the following description: "A short, natural language description of the station as it may be described by a person providing directions. In the least, it should include the station_name, if there is one, or if the station goes by multiple names colloquially, all these names should be included. The description may also include the address of the station, the operating agency, and the modes of transit that operate in this station."
Just my thoughts.

Reply:
@uwtcat yes, that's a much better description. In general, all the 'description's are very very generic. I focused on the structure first and was planning to go back and clean up grammar and improve descriptions. I like your words, thanks!